module/metrics/cache.go

```diff
@@ -33,11 +34,19 @@ func NewCacheCollector(chain flow.ChainID) *CacheCollector {
 			ConstLabels: prometheus.Labels{LabelChain: chain.String()},
 		}, []string{LabelResource}),
 
+		notfounds: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name:        "notfounds_total",
+			Namespace:   namespaceStorage,
+			Subsystem:   subsystemCache,
+			Help:        "the number of times the queried item was not found in either cache or database",
+			ConstLabels: prometheus.Labels{LabelChain: chain.String()},
```

Comment:
Do you know why we use the `chain` label here? Seems unrelated to the storage caches. 🤔